mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/bridge/PropertyBridge.java

```diff
@@ -52,11 +53,12 @@
 	 * Reading from the {@link PojoElement} should be done using
 	 * {@link org.hibernate.search.mapper.pojo.model.PojoModelElementAccessor}s retrieved when the
 	 * {@link #bind(PropertyBridgeBindingContext)} method was called.
-	 *
-	 * @param target The {@link DocumentElement} to write to.
+	 *  @param target The {@link DocumentElement} to write to.
 	 * @param source The {@link PojoElement} to read from.
+	 * @param sessionContext A {@link SessionContext} that can be {@link SessionContext#unwrap(Class) unwrapped}
+	 * to a more useful type, such as a Hibernate ORM Session (if using the Hibernate ORM mapper).
```

Comment:
If we take into account my comment about session unwrapping, we will need to update these comments.